resources/qml/TimelineView.qml

```diff
@@ -116,19 +116,10 @@ Page {
 			boundsBehavior: Flickable.StopAtBounds
 			pixelAligned: true
 
-			MouseArea {
-				anchors.fill: parent
-				acceptedButtons: Qt.NoButton
```

Comment:
Can you add that to the ScrollHelper? That way you can still flick the timeline by grabbing it. At least I like that feature, not sure how others feel about it though.